test/example.md

```diff
@@ -167,6 +167,17 @@ callAsyncFunction((a, b) => {
 });
 ```
 
+## functionWithFunction(fn, Returns:, number)
```

Comment:
`## functionWithFunction(fn, Returns:, number)` this doesn't look correct, is it?